views/administration/school.hbs

```diff
@@ -284,9 +284,28 @@
                                             {{/userHasPermission}}
                                         {{/userHasPermission}}
                                     />
-                                    {{$t "administration.school.label.createChatRoom" }}
+                                    {{$t "administration.school.label.messengerSchoolRoom" }}
                                     <p class="text-muted">
-                                    {{$t "administration.school.text.chatRoomPermissions" }}
+                                    {{$t "administration.school.text.messengerSchoolRoom" }}
+                                    </p>
+                                </label>
+                            </div>
+							<div class="form-group {{#ifConfig "MATRIX_MESSENGER__SCHOOL_SETTINGS_VISIBLE" false}}hidden{{/ifConfig}} {{#ifConfig "FEATURE_MESSENGER_ONETOONE_OPTION_VISIBLE" false}}hidden{{/ifConfig}}">
```

Comment:
I can't find the feature flag `FEATURE_MESSENGER_ONETOONE_OPTION_VISIBLE` in the schema.